Cropping faces from image/Requirements.txt

```diff
@@ -0,0 +1,5 @@
+Firstly, Make sure you have Python installed in your Machine. 
+Second requirement is the installation of OpenCV in your PC. You can install it using 'pip install opencv-python' if not installed.
+For the Haar cascades to work,user needsto download the trained classifier XML file (haarcascade_frontalface_default.xml), which is available in OpenCv’s GitHub repository. 
+Save this file to your working location.
```

Comment:
This is not the way of defining the requirements.txt file. Take a look at this file:
https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Amazon-Price-Alert/requirements.txt

Reply:
Ok will do these changes asap.